### PR TITLE
chore: add plugin test

### PR DESCRIPTION
**Pull request by starsz**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [ ] Bugfix
- [ ] New feature provided
- [ ] Improve performance
- [ ] Backport patches

- Related issues

___
Add plugin e2e test that we don't have before.
image

### CODE REVIEW

File: api/test/e2enew/plugin/plugin_suite_test.go [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+ * limitations under the License.
+ */
+package ssl
+
+import (
+	"github.com/onsi/ginkgo"
+	"testing"
+)
+
+func TestRoute(t *testing.T) {
```
Comment (nic-chen) [inline api/test/e2enew/plugin/plugin_suite_test.go:24]: need to update the function name

File: api/test/e2enew/plugin/plugin_test.go [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package ssl
+
+import (
+	"e2enew/base"
+	"github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/extensions/table"
+	"net/http"
```
Comment (nic-chen) [inline api/test/e2enew/plugin/plugin_test.go:23]: style

File: api/test/e2enew/plugin/plugin_suite_test.go (hunk 1 above)
Comment (starsz) [reply api/test/e2enew/plugin/plugin_suite_test.go:24]: Oh no.My fault.

File: api/test/e2enew/plugin/plugin_test.go (hunk 2 above)
Comment (starsz) [reply api/test/e2enew/plugin/plugin_test.go:23]: Go it.

**Review by nic-chen — APPROVED** [review]

**Review by tokers — APPROVED** [review]

**Pull request merged**